access/ms-teams/README.md

```diff
@@ -175,4 +175,54 @@ access request.
 Select `Deny` and verify that the request was indeed denied using
 `tctl request ls`.
 
-Specify ```preload = true``` at the top of your configuration file if you want plugin to check the user existence prior to startup.
\ No newline at end of file
+Specify ```preload = true``` at the top of your configuration file if you want plugin to check the user existence prior to startup.
+
+## Plugin-specific Architecture
+
+The plugin leverages 2 different Microsoft APIs to provide access-request notifications:
+```mermaid
```

Comment:
I'm not sure I'm getting the correspondence between this diagram and the text below. If I understand it correctly, we use the graphAPI to install the plugin into Teams, and then use the Bot Service to actually send the messages. If that's the case, then it might be better presented as a sequence diagram. For example, something like: 

```mermaid
sequenceDiagram
    participant Teleport    
    participant Plugin
    participant GraphAPI
    participant BotService as Azure Bot Service
    participant MsTeams
    actor Users

    Note over Plugin,MsTeams: Azure Tenant
    Plugin->>GraphAPI: Install TeamsApp with AppID/Secret
    activate GraphAPI
    GraphAPI->>MsTeams: Install Process
    activate MsTeams
    MsTeams-->>GraphAPI: OK
    deactivate MsTeams
    GraphAPI-->>Plugin: OK
    deactivate GraphAPI

    loop forever
        Teleport->>Plugin: Send Events
        Plugin->>BotService: Message with AppID/Secret
        BotService->>MsTeams: PostMessage as TeamsAppID
        MsTeams->>Users: Message
    end
```

Reply:
It's way clearer, thanks.